content/coverpage.tex

```diff
@@ -47,8 +47,60 @@ \section*{Sponsored by}
 \end{itemize}
 
 \section*{Authors and Collaborators}
-This document is a collaborative effort consisting of several releases of \openshmem versions 1.0 through 1.5. This section lists the authors and contributors in reverse chronological order, starting with \openshmem 1.5.
+This document is a collaborative effort consisting of several releases of \openshmem versions 1.0 through 1.6. This section lists the authors and contributors in reverse chronological order, starting with \openshmem 1.6.
 
+\subsection*{\openshmem 1.6}
+\begin{multicols}{2}
+\begin{itemize}
+\setlength\itemsep{0.1em}
+\item Ferrol Aderholdt, NVIDIA
+\item Muhammad Awad, \ac{AMD}
+\item Matthew Baker, \ac{ORNL}
+\item Swen Boehm, \ac{ORNL}
+\item Aurelien Bouteiller, \ac{UTK}
+\item Mark Brown, Intel
+\item Bob Cernohous, \ac{HPE}
+\item James Dinan\footnotemark[1], NVIDIA
+\item Megan Grodowitz, Arm Inc.
+\item Max Grossman, Georgia Tech
```

Comment:
@agrippa Is this the right institution for you?

Reply:
I believe @agrippa  is currently affiliated with Cruise (https://www.getcruise.com/), but he represented Georgia Tech when he served on the spec committee.

Reply:
Thanks! Max, let us know if you want to update your affiliation.